students/douglas_klos/session3/list_lab.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+
+# Douglas Klos
+# Python 210, Exercise 3
+# January 24st, 2019
+# list_lab.py
+
+
+def series1():
+    """
+    Creates the specified list and displays it. Asks user for another
+    fruit and adds it to end of list.  Displays list again.  Asks user
+    for a number, prints it back and its coressponding fruit.
+    Adds another fruit to the beginning of list using +.
+    Adds another fruit to the beginning of list using insert.
+    Displays all fruit in list beginning with P.
+    """
+
+    print("Series 1:")
+    fruit = ["Apples", "Pears", "Oranges", "Peaches"]
+    print(fruit)
+
+    # Ask a user for a fruit and add to end of list.
+    fruit += (input("Enter the name of fruit to be added: "),)
+    print(fruit)
+
+    # Ask user for anumber and display the number back to the use and
+    # the fruit corresponding to that number.  No error trapping.
+    index = input("Enter the index of fruit to print: ")
+    print("index = " + index + ", fruit = " + fruit[int(index)-1])
```

Comment:
consider using an `f` string instead of using `+` for creating strings
`print(f"index = {index}, fruit = {fruit[int(index) - 1]}")`  Makes it a little more readable